plugins/kubernetes-backend/api-report.md

```diff
@@ -101,12 +102,6 @@ export interface CustomResource extends ObjectToFetch {
   objectType: 'customresources';
 }
 
-// @public (undocumented)
-export interface CustomResourcesByEntity extends KubernetesObjectsByEntity {
```

Comment:
Think that you're going to want to re-export these things from the `-node` package here, otherwise this is a breaking change and will need to be updated in the changeset with some migration docs for what end-users need to do to fix the errors that they might have. Also will need updating to `minor` if you're wanting to go that route /cc @jamieklassen @mclarke47

Reply:
Thanks Ben for your feedback. We will re-export them from the -node package to avoid a breaking change